camp/MHerrman/Mateo_Xbox_Controller_2_Buttons/src/main/java/frc/robot/commands/MotorBackward.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.FeederSubsystem;
+
+public class MotorBackward extends Command{
+    private FeederSubsystem feeder;
+
+    public MotorBackward(FeederSubsystem feeder){
+        this.feeder = feeder;
+        addRequirements(feeder);
+    }
+
+    public void initialize(){
```

Comment:
If you don't have anything to run in initialize, you should not have it in this class. the superclass initialize will run.